build/helper/documentation_helper.py

```diff
@@ -888,6 +890,32 @@ def get_attribute_repeated_caps_with_conjunction(attr):
     return caps
 
 
+# Arbitrary set of example attributes that support a given repeated capability type, for a given module. Used to customize the doc tips of each attribute
+_example_repeated_cap_attributes = {
```

Comment:
I think a better place for this would be the rep caps dict in config metadata - https://github.com/ni/nimi-python/blob/2844f476a46eed05fa140685ec7c1c2f02d63015/src/nidcpower/metadata/config.py#L41

Reply:
That's a good idea. I've moved this info to the metadata:
1. [PR](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/163585) for Git-based APIs
2. [PR](https://ni.visualstudio.com/DefaultCollection/DevCentral/_git/shelves/pullrequest/163591) for perforce-based APIs